src/libraries/System.Private.Xml/src/System/Xml/Core/XmlEncodedRawTextWriterAsync.cs

```diff
@@ -688,13 +688,13 @@ private async Task EncodeCharsAsync(int startOffset, int endOffset, bool writeAl
                 bufBytesUsed += bEnc;
                 if (bufBytesUsed >= (bufBytes.Length - 16))
                 {
-                    await stream.WriteAsync(bufBytes, 0, bufBytesUsed).ConfigureAwait(false);
+                    await stream.WriteAsync(bufBytes.AsMemory(0, bufBytesUsed)).ConfigureAwait(false);
```

Comment:
This is generated file, please update `XmlRawTextWriterGeneratorAsync.ttinclude` as well

Reply:
Thanks, I didn't know about the generated files. I wonder if this is the reason why the CI is failing on these unit tests. I'll make the changes and then reply back with my findings.

Reply:
I don't think the test failed from template, but the failures seems happening on the test i added recently, i will take a look what is wrong

Reply:
Please add mock overloads to the test stream so that it wouldn't call  `MemoryStream.WriteAsync(ReadOnlyMemory<byte>, CancellationToken)`, the details sent by PM
```csharp
public override ValueTask WriteAsync(ReadOnlyMemory<byte> source, CancellationToken cancellationToken = default)
{
    return default;
}
```